packages/dds/tree/src/tableSchema.ts

```diff
@@ -389,7 +389,7 @@ export namespace System_TableSchema {
 				columnOrId: TableSchema.Column<TCellSchema> | string,
 			): CellValueType | undefined {
 				const columnId = typeof columnOrId === "string" ? columnOrId : columnOrId.id;
-				return this.cells.get(columnId) as CellValueType | undefined;
+				return this.cells[columnId];
```

Comment:
Might be worth a comment here:

```suggestion
				// Unlike most objects, RecordNodes don't have the default inherited object properties, so this is safe
				return this.cells[columnId];
```